[libksba] Unable to build

```
checking for GPG Error - version >= 1.8... no
configure: error: libgpg-error is needed.
                See ftp://ftp.gnupg.org/gcrypt/libgpg-error/ .
libksba failed to build: `./configure --prefix=/usr/local` exited with 1
```
Missing `libgpgerror` dependency?

--- a/packages/libarchive.rb
+++ b/packages/libarchive.rb
@@ -7,7 +7,18 @@ class Libarchive < Package
   source_url 'http://www.libarchive.org/downloads/libarchive-3.3.2.tar.gz'
   source_sha256 'ed2dbd6954792b2c054ccf8ec4b330a54b85904a80cef477a1c74643ddafa0ce'
 
-
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libarchive-3.3.2-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libarchive-3.3.2-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libarchive-3.3.2-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libarchive-3.3.2-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: '06a0fd7aeaa505d75963286eb61302915b421a824aeec0237f4d6efed4f4f671',
+     armv7l: '06a0fd7aeaa505d75963286eb61302915b421a824aeec0237f4d6efed4f4f671',
+       i686: 'c902c5e75b4a7de6bc027fe971bb963957ed752e79dc6306734b2c1f3b0bab5b',
+     x86_64: '55d1f9d949edad46042e0a268c19cebc2398fc8711c99b1071e089d7588ccfef',
+  })
 
   depends_on 'acl'
   depends_on 'attr'
@@ -17,8 +28,7 @@ class Libarchive < Package
   def self.build
     system "./configure \
           --prefix=#{CREW_PREFIX} \
-          --libdir=#{CREW_LIB_PREFIX} \    
-          --disable-static"
+          --libdir=#{CREW_LIB_PREFIX}"
     system "make"
   end
 

--- a/packages/libass.rb
+++ b/packages/libass.rb
@@ -7,6 +7,19 @@ class Libass < Package
   source_url 'https://github.com/libass/libass/releases/download/0.14.0/libass-0.14.0.tar.xz'
   source_sha256 '881f2382af48aead75b7a0e02e65d88c5ebd369fe46bc77d9270a94aa8fd38a2'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libass-0.14.0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libass-0.14.0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libass-0.14.0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libass-0.14.0-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: '1749b98d9fcd1783cc6d04543ace006637732db0c0dab128597f86f122a1a28a',
+     armv7l: '1749b98d9fcd1783cc6d04543ace006637732db0c0dab128597f86f122a1a28a',
+       i686: '6a3a7609356d407f7d2b6f4e0e9beedb89bf37bd5113a626ebe7e703d042caa1',
+     x86_64: '5d4f11533ac29592c4af827480bed3609ee8d4ee48e3b34bb3b68934da7e1248',
+  })
+
   #depends_on 'automake' => :build
   #depends_on 'autoconf' => :build
   #depends_on 'libtool' => :build
@@ -14,7 +27,6 @@ class Libass < Package
   depends_on 'fontconfig'  
   
   def self.build
-    system "./autogen.sh"
     system "autoconf"
     system "./configure --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX}"
     system "make"

--- a/packages/libedit.rb
+++ b/packages/libedit.rb
@@ -7,12 +7,24 @@ class Libedit < Package
   source_url 'https://thrysoee.dk/editline/libedit-20170329-3.1.tar.gz'
   source_sha256 '91f2d90fbd2a048ff6dad7131d9a39e690fd8a8fd982a353f1333dd4017dd4be'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libedit-3.1-0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libedit-3.1-0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libedit-3.1-0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libedit-3.1-0-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: '90f9db2636bdcae5cb4c8d6a0d020398b13c6290164c27776507689bc5fc85be',
+     armv7l: '90f9db2636bdcae5cb4c8d6a0d020398b13c6290164c27776507689bc5fc85be',
+       i686: '0590ee8aa7923a29332aa436ff42b909cf968c7ab50121df43539d968e3c496f',
+     x86_64: '6f6ed8dd83aad5895b5e3927c673d66c5d676a345f3bc6913e94392a74db929a',
+  })
 
   def self.build
-    system "./configure  \
-	    --prefix=#{CREW_PREFIX} \
-	    --libdir=#{CREW_LIB_PREFIX} \ 
-            CPPFLAGS=\"-I/usr/local/include/ncurses\""
+    system "./configure \
+            --prefix=#{CREW_PREFIX} \
+            --libdir=#{CREW_LIB_PREFIX} \
+            CPPFLAGS=\"-I#{CREW_PREFIX}/include/ncurses\""
     system "make"
   end
 

--- a/packages/libksba.rb
+++ b/packages/libksba.rb
@@ -7,7 +7,20 @@ class Libksba < Package
   source_url 'https://www.gnupg.org/ftp/gcrypt/libksba/libksba-1.3.5.tar.bz2'
   source_sha256 '41444fd7a6ff73a79ad9728f985e71c9ba8cd3e5e53358e70d5f066d35c1a340'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libksba-1.3.5-0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libksba-1.3.5-0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libksba-1.3.5-0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libksba-1.3.5-0-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: '9992a30c1ec084d843db72b5d183f46e65596ab52925b1807b1c7476dfa0f071',
+     armv7l: '9992a30c1ec084d843db72b5d183f46e65596ab52925b1807b1c7476dfa0f071',
+       i686: 'e69de6e5e76ae483bc8c350eb3ad9ab9cc1f03b1680c05bef2ea2acb4a4dc993',
+     x86_64: '4c0a4853f87105315ae990d0ccc6450bab22dd40666a92d0a22985b301739833',
+  })
 
+  depends_on 'libgpgerror'
   depends_on 'npth'
 
   def self.build

--- a/packages/libnl3.rb
+++ b/packages/libnl3.rb
@@ -7,6 +7,18 @@ class Libnl3 < Package
   source_url 'https://www.infradead.org/~tgr/libnl/files/libnl-3.2.25.tar.gz'
   source_sha256 '8beb7590674957b931de6b7f81c530b85dc7c1ad8fbda015398bc1e8d1ce8ec5'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libnl3-3.2.25-0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libnl3-3.2.25-0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libnl3-3.2.25-0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libnl3-3.2.25-0-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: 'fdce1a46f9a89772ef9b51dd5bf738a24966908027299f5c3dc049c95b8f2a7b',
+     armv7l: 'fdce1a46f9a89772ef9b51dd5bf738a24966908027299f5c3dc049c95b8f2a7b',
+       i686: '1efa9cd106f16ff347d82bb3cb0734d8393d53edf51db2d294e9788a20036bb4',
+     x86_64: '4494bd6d1919090838d86f75d6cc1ea86becc467400d58dffc4e905d27a373da',
+  })
 
   def self.build
 	system "./configure \

--- a/packages/libopencoreamr.rb
+++ b/packages/libopencoreamr.rb
@@ -7,7 +7,18 @@ class Libopencoreamr < Package
   source_url 'https://downloads.sourceforge.net/project/opencore-amr/opencore-amr/opencore-amr-0.1.5.tar.gz'
   source_sha256 '2c006cb9d5f651bfb5e60156dbff6af3c9d35c7bbcc9015308c0aff1e14cd341'
 
-
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libopencoreamr-0.1.5-0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libopencoreamr-0.1.5-0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libopencoreamr-0.1.5-0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libopencoreamr-0.1.5-0-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: '637a39ca077305a37b9a7e58e46d4344f29d5ce1e1ab6febd7d93d6727ef88bd',
+     armv7l: '637a39ca077305a37b9a7e58e46d4344f29d5ce1e1ab6febd7d93d6727ef88bd',
+       i686: '75b3d8849bac7849ab4ea916ec94e5d05870becf98bc29d007a18fdf58ea732a',
+     x86_64: '22c116cc561c16012f909768f1794d27b72335573c1ac97a67d4463693be492a',
+  })
 
   def self.build
     system "./configure --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX}"

--- a/packages/libopus.rb
+++ b/packages/libopus.rb
@@ -7,7 +7,18 @@ class Libopus < Package
   source_url 'https://archive.mozilla.org/pub/opus/opus-1.2.1.tar.gz'
   source_sha256 'cfafd339ccd9c5ef8d6ab15d7e1a412c054bf4cb4ecbbbcc78c12ef2def70732'
 
-
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libopus-1.2.1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libopus-1.2.1-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libopus-1.2.1-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libopus-1.2.1-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: '7929d1a7b49da919aedd9b3e2fe8cd7a221535d93c35a01dfd561aa449365dbb',
+     armv7l: '7929d1a7b49da919aedd9b3e2fe8cd7a221535d93c35a01dfd561aa449365dbb',
+       i686: '75fb863728da71393412626a5944bfac8a716e6d938eb3ee36d3b080318fc577',
+     x86_64: '317439fc79b79ff47626a5c5da9a7a86c55711960dddc8d8f89f3226b4d6fbbf',
+  })
 
   def self.build
     system "./configure  \

--- a/packages/libpcap.rb
+++ b/packages/libpcap.rb
@@ -7,7 +7,18 @@ class Libpcap < Package
   source_url 'https://www.tcpdump.org/release/libpcap-1.8.1.tar.gz'
   source_sha256 '673dbc69fdc3f5a86fb5759ab19899039a8e5e6c631749e48dcd9c6f0c83541e'
 
-
+  binary_url ({
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libpcap-1.8.1-0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libpcap-1.8.1-0-chromeos-x86_64.tar.xz',
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libpcap-1.8.1-0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libpcap-1.8.1-0-chromeos-armv7l.tar.xz',
+  })
+  binary_sha256 ({
+       i686: '6b0bdf1e1e244d3eac8ade1a2ba5f4442af97567a785638b6fc7619a06381465',
+     x86_64: 'f6b91e914ff739c6b17160da29566af9746b4c38ad7ff24394b43594d62eb8b6',
+    aarch64: 'not',
+     armv7l: 'not',
+  })
 
   def self.build
     system "./configure --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX}"

--- a/packages/libseccomp.rb
+++ b/packages/libseccomp.rb
@@ -7,11 +7,23 @@ class Libseccomp < Package
   source_url 'https://github.com/seccomp/libseccomp/archive/v2.3.3.tar.gz'
   source_sha256 '5a52495207f00d1254707f11226e17c16ec53f5038d65bbabf1892873fa2fe5b'
 
-
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libseccomp-2.3.3-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libseccomp-2.3.3-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libseccomp-2.3.3-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libseccomp-2.3.3-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: 'c91f75159f6f611093aa259edb509f235cb490c8789931f9e905b2e39b9f3609',
+     armv7l: 'c91f75159f6f611093aa259edb509f235cb490c8789931f9e905b2e39b9f3609',
+       i686: 'e244798d1d0cf120f925e0bece4f507bfed73e975aeb2c73b6910452ece85252',
+     x86_64: 'b7b39bcc650e34789b860a565cea3abda054a7f2755e5d8516d3574b612223cd',
+  })
 
   def self.build
+    system './autogen.sh'
     system "./configure --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX}"
-    system "make"
+    system 'make'
   end
 
   def self.install

--- a/packages/libtheora.rb
+++ b/packages/libtheora.rb
@@ -7,6 +7,18 @@ class Libtheora < Package
   source_url 'https://downloads.xiph.org/releases/theora/libtheora-1.1.1.tar.bz2'
   source_sha256 'b6ae1ee2fa3d42ac489287d3ec34c5885730b1296f0801ae577a35193d3affbc'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libtheora-1.1.1-0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libtheora-1.1.1-0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libtheora-1.1.1-0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libtheora-1.1.1-0-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: '8737530ad3a7923d32b273258123f8c18d9f7c7ef13be95880e9b4e85d435cfb',
+     armv7l: '8737530ad3a7923d32b273258123f8c18d9f7c7ef13be95880e9b4e85d435cfb',
+       i686: '971dfa07b59b68f7c4861b2bbfd65f6b64a937c69b8fd42bf4fa4b1b567ee49f',
+     x86_64: '281643b002a7da1589c01550d7a73ca0eed69d4bde2529b6c881bc8debd2bade',
+  })
 
   depends_on 'libvorbis'
   depends_on 'libsdl2'

--- a/packages/libuninum.rb
+++ b/packages/libuninum.rb
@@ -7,6 +7,18 @@ class Libuninum < Package
   source_url 'https://billposer.org/Software/Downloads/libuninum-2.7.tar.bz2'
   source_sha256 'e704983c3c00e9b76bd836b8b83ce31bfe4eb1752eee8be123cf97c1275076ea'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libuninum-2.7-0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libuninum-2.7-0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libuninum-2.7-0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libuninum-2.7-0-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: '8cca34b8e60fed8fd5f8506f4b91ababb6343fb89a268a03fb590ebf44f29cef',
+     armv7l: '8cca34b8e60fed8fd5f8506f4b91ababb6343fb89a268a03fb590ebf44f29cef',
+       i686: '36aad93b00c85623884776b89dab2dde5c7e8ef393b13b9faf4b175ea2787704',
+     x86_64: 'f2119fada66d21f242a33068c46267c1f05cdcb517dcbe76ab9476ca556e22cc',
+  })
 
   def self.build
     system "./configure --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX}"

--- a/packages/libvoamrwbenc.rb
+++ b/packages/libvoamrwbenc.rb
@@ -7,7 +7,18 @@ class Libvoamrwbenc < Package
   source_url 'https://downloads.sourceforge.net/project/opencore-amr/vo-amrwbenc/vo-amrwbenc-0.1.3.tar.gz'
   source_sha256 '5652b391e0f0e296417b841b02987d3fd33e6c0af342c69542cbb016a71d9d4e'
 
-
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libvoamrwbenc-0.1.3-0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libvoamrwbenc-0.1.3-0-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libvoamrwbenc-0.1.3-0-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libvoamrwbenc-0.1.3-0-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: '983297522b623ee92bc5886479b85636cd88ef5f67d36da4a4ea0b2374583cb7',
+     armv7l: '983297522b623ee92bc5886479b85636cd88ef5f67d36da4a4ea0b2374583cb7',
+       i686: 'bf93b9324bc99471aea25f4c2e36f24db682114a394e09c536fae23b95cd0b6d',
+     x86_64: '10da147ed2bd9987882cd7dcaf9a755c1e8395a9719a3caed66710d7d4e8d332',
+  })
 
   def self.build
     system "./configure --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX}"

--- a/packages/libwebp.rb
+++ b/packages/libwebp.rb
@@ -7,7 +7,18 @@ class Libwebp < Package
   source_url 'https://storage.googleapis.com/downloads.webmproject.org/releases/webp/libwebp-0.6.1.tar.gz'
   source_sha256 '06503c782d9f151baa325591c3579c68ed700ffc62d4f5a32feead0ff017d8ab'
 
-
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libwebp-0.6.1-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libwebp-0.6.1-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libwebp-0.6.1-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libwebp-0.6.1-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: '31c50e823a1a4c3738fc7b205a763653d03f6c567c23c624252d40f6b01fa25a',
+     armv7l: '31c50e823a1a4c3738fc7b205a763653d03f6c567c23c624252d40f6b01fa25a',
+       i686: '7ceece5c2f15e3d55bdc93426c048fa4fe2b9c425217f0cf03ded2380f0b8a1a',
+     x86_64: '749abdd3f587597de4aa19a0ebfc3a014d48c08e2304249e8b1533eee6e6fb67',
+  })
 
   def self.build
     system "./configure --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX}"

--- a/packages/libxvid.rb
+++ b/packages/libxvid.rb
@@ -7,6 +7,18 @@ class Libxvid < Package
   source_url 'https://downloads.xvid.com/downloads/xvidcore-1.3.5.tar.bz2'
   source_sha256 '7c20f279f9d8e89042e85465d2bcb1b3130ceb1ecec33d5448c4589d78f010b4'
 
+  binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/libxvid-1.3.5-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/libxvid-1.3.5-chromeos-armv7l.tar.xz',
+       i686: 'https://dl.bintray.com/chromebrew/chromebrew/libxvid-1.3.5-chromeos-i686.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/libxvid-1.3.5-chromeos-x86_64.tar.xz',
+  })
+  binary_sha256 ({
+    aarch64: '0587f20f1a2ea2ede14054ccddfc2fbef2a0ef5b611c3ba679c0d1724da365eb',
+     armv7l: '0587f20f1a2ea2ede14054ccddfc2fbef2a0ef5b611c3ba679c0d1724da365eb',
+       i686: '2d398811d6468a57dd236907d7b2e8a71263fba2423d3baf4d04f8f1b7ac5b1a',
+     x86_64: 'e990606cef5b83dcffa96e6595538240934c6f926ca49701c165322811f8a44f',
+  })
 
   depends_on 'yasm'
 

